site_scons/site_init.py

```diff
@@ -0,0 +1,25 @@
+"""init some daos specific functions functions"""
+import re
+import SCons
+from SCons.Scanner import FindPathDirs
+
+include_re = re.compile(r'^#include[ \t].(\S+).$', re.M)
+
+def c_scan(node, env, path):
+    """For whatever reason the default scanner isn't working in some cases."""
+    contents = node.get_text_contents()
+    includes = include_re.findall(contents)
+    results = []
+    for include in includes:
+        result = SCons.Node.FS.find_file(include, path)
+        if result is None:
```

Comment:
(pylint-no-else-continue) Unnecessary "else" after "continue", remove the "else" and de-indent the code inside it